### ISSUE TITLE
Write debug messages to stderr instead of stdout

### ISSUE THREAD

**Issue by varac** [LaunchPlatform/beanhub-cli#7]
First of all, thanks for the formatter, it's great because it preserves comments :tada: 

Please write all debug messages to `stderr` instead of `stdout`. I'd like to integrate `bh format --stdin-mode` as a formatter for my editor Neovim using [conform.nvim](https://github.com/stevearc/conform.nvim). But I noticed that in stdin-mode, the debug messages are sent to stdout as well as the resulting transaction, messing up my journal file.

# Steps to reproduce

Given this journal: 

```console
$ cat simple.beancount
1970-01-01 open Assets:Cash
1970-01-01 open Expenses:Etc

2021-12-31 * "me" "chocolate"
  Expenses:Etc                                               3 EUR
  Assets:Cash
```

When I run run the formatter in stdin-mode, redirecting stdout and stderr to two different files:

```console 
$ cat simple.beancount | bh format --stdin-mode > bh.stdout 2> bh.stderr

$ cat bh.stdout
[21:33:31] INFO     Processing in stdin mode                        format.py:71
[21:33:32] INFO     Calculate column width                      formatter.py:646
           INFO     Collecting                                  formatter.py:130
1970-01-01 open Assets:Cash
1970-01-01 open Expenses:Etc

2021-12-31 * "me" "chocolate"
  Expenses:Etc                                               3 EUR
  Assets:Cash
           INFO     done                                           format.py:108

$ cat bh.stderr 
$
```

**Comment by fangpenlin** [comment]
Hi @varac, thanks for reporting the issue. Yes, you're right, logs should be written to stderr instead of stdout. I've already addressed the problem by #8 and I just released [1.4.1](https://github.com/LaunchPlatform/beanhub-cli/releases/tag/1.4.1).

Please let me know if this addresses your problems.

**Comment by varac** [comment]
Hi @fangpenlin, thank you for the fast fix! I confirm it's working as it should now, and created a PR to [include beanhub-cli in the mason registry](https://github.com/mason-org/mason-registry/pull/7475).
